docs/content/preview/yugabyte-cloud/cloud-monitor/overview.md

```diff
@@ -194,3 +164,52 @@ DocDB uses a highly customized version of [RocksDB](http://rocksdb.org/), a log-
+| YB-TServer RocksDB Cache Usage (GB) | A block requires multiple touches before it is added to the multi-touch (hot) portion of the cache.<br><br>Multi Touch: the size (in bytes) of the cache usage by blocks having multiple touches.<br>Single Touch: the size (in bytes) of the cache usage by blocks having only a single touch. | |
+| YB-TServer RocksDB Blooms | Checked: the number of times the bloom filter has been checked.<br>Useful: the number of times the bloom filter has avoided file reads (avoiding iops). | Bloom filters are hash tables used to determine if a given SSTable has the data for a query looking for a particular value.<br>Bloom filters are not helpful for range queries. |
+| YB-TServer RocksDB Stall Time (ms) | Time the writer has to wait for compactions or flushes to finish. | |
+| YB-TServer RocksDB Write Rejections | The number of RPC requests rejected due to the number of majority SST files. | Rejections can happen due to hitting the soft limit on SST files. |
+| YB-TServer RocksDB Memory Pressure Rejections | The number of RPC requests rejected due to the number of majority SST files. | Rejections can happen due to hitting the soft memory limit (set to 85% of the hard limit), and so on. |
+| YB-TServer RocksDB Flush Write Bytes (MiB) | The number of bytes written during the flush process. | |
+| YB-TServer RocksDB Compaction Files | The average number of files in any single compaction. | |
+| YB-TServer RocksDB Compaction Time (ms) | Time for the compaction processes to complete. | |
+| YB-TServer RocksDB Transactions | Expired: the number of expired distributed transactions.<br>Conflicts: the number of conflicts detected among uncommitted distributed transactions.<br><br>This is related to the process that resolves conflicts for write transactions. This process reads all intents that could conflict and tries to abort transactions with a lower priority. If a write transaction conflicts with a higher-priority transaction, then an error is returned and this metric is iterated. | |
+| YB-TServer RocksDB Transaction Pool Cache | Percentage of transaction pool requests fulfilled by the transaction pool cache. | |
+| YB-TServer RocksDB Tablet Splitting Operations | Tracks tablet splits that have passed each of the following states:<br/>
+Add: Successfully submitted split RPC requests (applied on leader only)<br/>
+Apply: Successfully applied Raft split operations.<br/>
+Post Split Compactions: Number of successfully started post-split compactions. | If these metrics are high, a lot of tablet splits are being performed. These numbers should be roughly equal, and any large gap between them is anomalous. |
+| YB-TServer RocksDB Tablet Splitting Manager Runtime (ms) | How long the last automatic split manager run took. | The automatic split manager runs periodically to determine whether any new tablets are eligible for splitting. |
```

Comment:
How long the last automatic split manager run took.  >  The duration of time the last automatic split manager run took.